velox/docs/functions/presto/conversion.rst

```diff
@@ -367,7 +367,8 @@ Valid examples
 From VARCHAR
 ^^^^^^^^^^^^
 
-There is a set of strings allowed to be casted to boolean. Casting from other strings to boolean throws.
+There is a set of strings allowed to be casted to boolean. These strings are `t, f, 1, 0, true, false` and their upper case equivalents.
```

Comment:
nit: This seems a little wordy, what about just

The strings `t, f, 1, 0, true, false` and their upper case equivalents are allowed to be casted to boolean. Casting from other strings to boolean throws.